src/main/java/com/yurjinia/project_structure/comment/contreller/request/CreateCommentRequest.java

```diff
@@ -7,4 +7,5 @@
 public class CreateCommentRequest {
     @Size(max = 1024)
     private String text;
+    private String timeZone;
```

Comment:
add annotation for checking if the field is not blank. we don't want to create a comment without knowing the timezone of the user.